bindings/python/gnucash_core.i

```diff
@@ -44,6 +44,29 @@
 %feature("autodoc", "1");
 %module(package="gnucash") gnucash_core_c
 
+%typemap(in) QofPercentageFunc {
+  if ($input == Py_None) {
+    $1 = (QofPercentageFunc)0;
+  } else {
+    $1 = (QofPercentageFunc)PyLong_AsVoidPtr($input);
+  }
+}
+
+%pythoncode
+%{
+import ctypes
+
+qof_percentage_func_type = ctypes.CFUNCTYPE(None, ctypes.c_char_p, ctypes.c_double)
```

Comment:
IIUC this is a type so according to our [style guide](https://wiki.gnucash.org/wiki/CodingStandard) it should be camel-case.

Reply:
Should be cured by e7e3b55358a663913e7989bd7569b469f92b648d